README.md

```diff
@@ -54,9 +54,26 @@ import CryptorECC
 
 #### Elliptic curve private key
 
-Generate your elliptic curve private key using a third party provider:
+you can generate an ECPrivate key using BlueECC.
 
-You can generate a `p-256` private key as a `.p8` file for Apple services from [https://developer.apple.com/account/ios/authkey](https://developer.apple.com/account/ios/authkey/). This will produce a key that should be formatted as follows:
+```swift
+let p256PrivateKey = try ECPrivateKey(for: .prime256v1)
```

Comment:
Rather than an initializer, I'd use a static factory function here to return a new instance.

The [Swift API design guidelines](https://swift.org/documentation/api-design-guidelines/) suggest that `make()` is the appropriate keyword.  So something like:
```swift
public static func make(for curve: EllipticCurve) throws -> ECPrivateKey
```